### PR TITLE
feat: add Video Stitcher slate code samples and tests

### PR DESCRIPTION
**Pull request by irataxy**
Fixes 202972969

1/3 parts

> It's a good idea to open an issue first for discussion.

- [X] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [X] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [X] API's need to be enabled to test (tell us) - Video Stitcher (allowlisted) - contact nickcook@
- [ ] Environment Variables need to be set (ask us to set them)
- [X] **Tests** pass:   `mvn clean verify` **required**
- [X] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.io.IOException;
+
+public class CreateSlate {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String slateId = "my-slate-id";
+    String slateUri = "my-slate-uri"; // URI of an MP4 video with at least one audio track
```
Comment (averikitsch) [inline media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java:34]: Should this example string include a file extension like `.mp4`?

File: media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+
+  public static void createSlate(String projectId, String location, String slateId, String slateUri)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (VideoStitcherServiceClient videoStitcherServiceClient =
+        VideoStitcherServiceClient.create()) {
+      var createSlateRequest =
```
Comment (averikitsch) [inline media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java:46]: We recommend not using var, see https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#java-11-features

File: media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          CreateSlateRequest.newBuilder()
+              .setParent(LocationName.of(projectId, location).toString())
+              .setSlateId(slateId)
+              .setSlate(Slate.newBuilder().setUri(slateUri).build())
+              .build();
+
+      Slate response = videoStitcherServiceClient.createSlate(createSlateRequest);
+      System.out.println("Slate: " + response.getName());
```
Comment (averikitsch) [inline media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java:54]: nit: You could print a more informative message

File: media/stitcher/src/main/java/com/example/stitcher/UpdateSlate.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    try (VideoStitcherServiceClient videoStitcherServiceClient =
+        VideoStitcherServiceClient.create()) {
+      var updateSlateRequest =
+          UpdateSlateRequest.newBuilder()
+              .setSlate(
+                  Slate.newBuilder()
+                      .setName(SlateName.of(projectId, location, slateId).toString())
+                      .setUri(slateUri)
+                      .build())
+              .setUpdateMask(FieldMask.newBuilder().addPaths("uri").build())
```
Comment (averikitsch) [inline media/stitcher/src/main/java/com/example/stitcher/UpdateSlate.java:54]: nit: a comment on what this is doing would be helpful. FieldMasks are very confusing to our users.

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java (hunk 1 above)
Comment (irataxy) [reply media/stitcher/src/main/java/com/example/stitcher/CreateSlate.java]: Done

**Review by averikitsch — APPROVED** [review]

**Pull request merged**